src/lib/mongoBackend/mongoQueryTypes.h

```diff
@@ -56,7 +59,8 @@ extern HttpStatusCode mongoEntityTypes
   const std::vector<std::string>&      servicePathV,
   std::map<std::string, std::string>&  uriParams,
   const std::string&                   apiVersion,
-  unsigned int*                        totalTypesP
+  unsigned int*                        totalTypesP,
+  bool                                 noAttrDetail = false
```

Comment:
I'd suggest not using default for `noAttrDetails` (and take the opportunity to remove also in apiVersion in the next function). The number of calls to this functions is small and that way we can ensure that it is correctly set in NGSIv1 calls (that are a bit "critial" due to its usage from IoTP).